develop/reference/articles/20-item-selector-criterion-and-return-types.markdown

```diff
@@ -1,64 +1,64 @@
         Image file entity type.
 
-    -   [AudioItemSelectorCriterion](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/collaboration/item-selector/com.liferay.item.selector.criteria.api/com/liferay/item/selector/criteria/audio/criterion/AudioItemSelectorCriterion.html):
+    -   [AudioItemSelectorCriterion](@app-ref@/collaboration/latest/javadocs/com/liferay/item/selector/criteria/audio/criterion/AudioItemSelectorCriterion.html):
         Audio file entity type.
 
-    -   [FileItemSelectorCriterion](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/collaboration/item-selector/com.liferay.item.selector.criteria.api/com/liferay/item/selector/criteria/file/criterion/FileItemSelectorCriterion.html):
+    -   [FileItemSelectorCriterion](@app-ref@/collaboration/latest/javadocs/com/liferay/item/selector/criteria/criteria/file/criterion/FileItemSelectorCriterion.html):
         Document Library file entity type.
 
-    -   [UploadItemSelectorCriterion](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/collaboration/item-selector/com.liferay.item.selector.criteria.api/com/liferay/item/selector/criteria/upload/criterion/UploadItemSelectorCriterion.html):
+    -   [UploadItemSelectorCriterion](@app-ref@/collaboration/latest/javadocs/com/liferay/item/selector/criteria/upload/criterion/UploadItemSelectorCriterion.html):
         Uploadable file entity type.
 
-    -   [URLItemSelectorCriterion](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/collaboration/item-selector/com.liferay.item.selector.criteria.api/com/liferay/item/selector/criteria/url/criterion/URLItemSelectorCriterion.html):
+    -   [URLItemSelectorCriterion](@app-ref@/collaboration/latest/javadocs/com/liferay/item/selector/criteria/url/criterion/URLItemSelectorCriterion.html):
         URL entity type.
 
-    -   [VideoItemSelectorCriterion](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/collaboration/item-selector/com.liferay.item.selector.criteria.api/com/liferay/item/selector/criteria/video/criterion/VideoItemSelectorCriterion.html):
+    -   [VideoItemSelectorCriterion](@app-ref@/collaboration/latest/javadocs/com/liferay/item/selector/criteria/video/criterion/VideoItemSelectorCriterion.html):
         Video file entity type.
 
--   [`com.liferay.wiki.api`](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/collaboration/wiki/com.liferay.wiki.api/) has wiki criterion.
+-   [`com.liferay.wiki.api`](@app-ref@/collaboration/latest/javadocs/com/liferay/wiki/item/selector/criterion/package-summary.html) has wiki criterion.
  
 **Web Experience App Suite Modules:**
 
--   [`com.liferay.site.item.selector.api`](https://docs.liferay.com/portal/7.0/javadocs/modules/apps/web-experience/site/com.liferay.site.item.selector.api/) has site criterion. 
+-   [`com.liferay.site.item.selector.api`](@app-ref@/web-experience/latest/javadocs/com/liferay/site/item/selector/criterion/package-summary.html) has site criterion. 
```

Comment:
@cody.hoag I updated the web-experience links in this doc, since I had the other collab links in here as well.